runelite-client/src/main/java/net/runelite/client/plugins/groundmarkers/GroundMarkerConfig.java

```diff
@@ -78,11 +82,11 @@ default amount getAmount()
 	@ConfigItem(
 		position = 1,
 		keyName = "markerColor",
-		name = "Default Marked tile Color",
```

Comment:
Color is capitalized here in the names but nowhere else. nvm im not gonna start a review from this lmao who gives a fuck